contents/blog/aruba-hackathon.md

```diff
@@ -0,0 +1,177 @@
+
+![dashboard templates](../images/blog/aruba/templates.png)
+
+These dashboards are accessible from the 'New Dashboard' modal in PostHog. Some templates require custom events, which you'll be asked to configure before creating the dashboard – you can also change the events you track later.
+
+![dashboard templates](../images/blog/aruba/setup-events.png)
+
+Got a request for a dashboard? DM someone from the marketing on our community Slack.
+
+Joe Martin also worked with Eli from the Website & Docs team to ship [PostHog Tracks](/tracks), a new way to discover tutorials organized by role.
+
+## A built-in data warehouse for PostHog
+
+- **Team DataBeach:** Marius Andra, Frank Hamand and Harry Waye
+
+Dubbed DataBeach because Frank and Harry started building the feature while sipping Piña Coladas by the beach, DataBeach is all about our long-term vision of  [simplifying the modern data stack](/blog/modern-data-stack-sucks) for startups. Less time spent wrangling data = more time shipping products. 
+
+![databeach](../images/blog/aruba/data-beach.png)
+
+The MVP they built consists of custom tables that are created and queried through the PostHog UI and API. These tables provide a way to store and query data from sources like Stripe, Hubspot (see above), Intercom, and more, along with data from PostHog.
+
+There's much to build before PostHog as a data warehouse is ready for you, but it's coming. Keep an eye on our [public roadmap](/roadmap) for updates.
+
+## Building a Zendesk killer in PostHog
+
+- **Team Arubug:** Tiina Turban, Simon Fisher, Paul D'Ambra, Cameron DeLeone, and Cory Watilo
+
+Last year, we adopted Zendesk to manage support for PostHog, but we don't love it. It's hard to prioritize tickets, and most lack the context a support hero or engineer needs to fix it. Enter team Arubug, who decided there had to be a better way.
+
+// TODO: Video of site app
+
+The team started by [building a site app](/tutorials/build-site-app) for bug reports based on our existing [Feedback Widget](/apps/feedback-widget), which sends a `$bug_report` event to PostHog.
+
+// TODO: Screenshot of dashboard
+
+These reports feed into a dashboard that tracks bug reports, helping us to identity trends. Bugs can broken down as tables with relevant properties, and session replays, attached.
+
+// TODO: Screenshot of bug report table 
+
+The team also built a communication tab into bug reports, so support can send emails and leave notes on tickets with additional context without leaving the app. Every email is a ClickHouse event tied to the initial UUID of the report event, with emails (for now) sent and received via Zapier.
+
+// TODO: Screenshot of communication tab
+
+Long term, we're looking at whether this could replace Zendesk for support, and be launched as a beta product for PostHog users one day.
```

Comment:
```suggestion
After the hackathon we've reviewed the plan:
- we realised that there's another problem internally: due to the increased complexity of our product it’s become harder for a single engineer to provide good support for all of our products. Support hero’s also shield product teams from feedback about their product, which means it’s harder to iterate on a product. So we're going to use this in-app integration to split out to the more specific team helping us do triaging effectively.
- we've decided at this point not to continue the work on the communication tab and instead integrate better with Zendesk, that said the way we handle support can be used by customers too, there will be a blog post coming to explain this further.
```

^ I think we could write a great blog post about how to include info about session recordings etc into their support ticketing tool and take advantage of session recordings like we do and do analytics on their support tickets like we do.